packages/backend-core/src/users/db.ts

```diff
@@ -362,13 +363,13 @@ export class UserDB {
       newCreators.length,
       async () => {
         // create the promises array that will be called by bulkDocs
-        newUsers.forEach((user: any) => {
+        newUsers.forEach(user => {
           usersToSave.push(
             UserDB.buildUser(
               user,
               {
                 hashPassword: true,
-                requirePassword: user.requirePassword,
+                requirePassword: false,
```

Comment:
`requirePassword` never exists on any user type, so I don't know why this was here. It's also never referenced at all in the frontend, so we definitely don't mix this in to the user type before sending it up.